app/view/clip-list.js

```diff
@@ -5,13 +5,15 @@ import readline from 'readline';
 import Clip from './clip';
 import {ipcRenderer} from 'electron';
 import autobind from 'autobind-decorator';
-
+import setting from '../setting.json';
 @autobind
 export default class ClipList extends React.Component {
   constructor() {
     super();
     this.state = {
-      logs: []
+      logs: [],
+      changeClicked: false,
```

Comment:
I think 
changeClicked, enableServiceHook state
should located at app-component. And pass them to ClipList.